Float.TinCan.ActivityLibrary/ActivityLaunchCoordinator.cs

```diff
@@ -9,6 +9,9 @@
 using Float.TinCan.LocalLRSServer;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using Rg.Plugins.Popup.Extensions;
+using Rg.Plugins.Popup.Pages;
+using Rg.Plugins.Popup.Services;
```

Comment:
@EBusch Ideally we would avoid adding new dependencies to this library. We can do that by creating new overridable methods for handling presenting and dismissing of the downloading page.